### PR TITLE
14TeV triboson cards for upgrade studies

### PR DESCRIPTION
**Pull request by vdutta**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/WVAToLNu2jA_VisWorZ_4f_NLO/WWAToLNu2jA_4f_NLO_madspin_card.dat [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ #set BW_cut 15                # cut on how far the particle can be off-shell
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/WVAToLNu2jA_VisWorZ_4f_NLO/WWAToLNu2jA_4f_NLO_madspin_card.dat]: can you please cleanup unused parameters?

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/WWZJets/WWZJets_4f_NLO/WWZJets_4f_NLO_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#import model loop_sm-ckm
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/WWZJets/WWZJets_4f_NLO/WWZJets_4f_NLO_proc_card.dat]: cleanup unused import

**Comment by vdutta** [comment]
@perrozzi should be done!

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/WVAToLNu2jA_VisWorZ_4f_NLO/WWAToLNu2jA_4f_NLO_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,40 @@
+#************************************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/WVAToLNu2jA_VisWorZ_4f_NLO/WWAToLNu2jA_4f_NLO_proc_card.dat]: last thing: for consistency please remove the banners in proc_card files

**Pull request merged**